packages/flutter_tools/lib/src/runner/flutter_command.dart

```diff
@@ -795,7 +795,8 @@ abstract class FlutterCommand extends Command<void> {
   void _printDeprecationWarning() {
     if (deprecated) {
       globals.printStatus('$warningMark The "$name" command is deprecated and '
-          'will be removed in a future version of Flutter.');
+          'will be removed in a future version of Flutter. '
+          'Refer to https://flutter.dev/docs/development/tools/sdk/releases');
```

Comment:
Maybe: "See https://flutter.dev/docs/development/tools/sdk/releases for previous releases of flutter"

Reply:
Thank you for the feedback! Your version is better. Will use it instead of the initially chosen one.